javascript/javascript3/week3/visual/main.js

```diff
@@ -0,0 +1,30 @@
+moveElement(document.querySelector("li"), { x: 20, y: 300 }).then(() => {
+  console.log("Element has been moved");
+});
+
+async function translateOneByOne() {
+  const redCircle = document.querySelector(".marks").firstElementChild;
+  await moveElement(redCircle, {
+    x: 20 - parseInt(redCircle.style.left),
+    y: 300 - parseInt(redCircle.style.top),
+  }).then(() => {
+    console.log("Element has been moved: " + redCircle);
```

Comment:
Remember that when using `await` code that is executed after that will wait for the previous promise to return. In this case, that means, you can safely put the `console.log` directly after the `await` line instead of wrapping it in a `then` chain element. 🤓